src/ui/Header.tsx

```diff
@@ -90,9 +94,9 @@ export const HeaderWrapper: React.FC<ComponentProps<typeof Flex>> = ({
       px={8}
       borderBottom="1px solid"
       borderColor="gray.100"
-      backgroundColor={bgColor}
+      {...rest}
```

Comment:
Do we know exactly what might be passed into HeaderWrapper?  If so, would it make sense to just specify them?

Reply:
I'm basically just trying to allow any props that might be passed to the div element be passed along. This allows the consumer to use style props on it: `<HeaderWrapper bg="red">` or some other style prop as necessary. For now it looks like we only use `bg` but I basically wanted `<HeaderWrapper>` to function like a `<Flex>` except with some default styles.